src/Squidex.Domain.Apps.Entities/Assets/Edm/EdmModelBuilder.cs

```diff
@@ -0,0 +1,61 @@
+﻿// ==========================================================================
+//  EdmModelBuilder.cs
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex Group
+//  All rights reserved.
+// ==========================================================================
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.OData.Edm;
+using Squidex.Infrastructure;
+
+namespace Squidex.Domain.Apps.Entities.Assets.Edm
+{
+    public class EdmModelBuilder : CachingProviderBase
```

Comment:
There is no need to cache it, it will never change. We can just create a private readonly instance for that.